src/components/ClaimItem/ClaimItem.tsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import { Grid, IconButton, Paper, Tooltip, Typography } from '@mui/material';
```

Comment:
Import our Typography to display the correct font, the MUI variant will have the default MUI font.